tests/network/sriov.go

```diff
@@ -172,36 +172,28 @@ var _ = Describe("SRIOV", Serial, decorators.SRIOV, func() {
 			if vmi.Annotations == nil {
 				vmi.Annotations = make(map[string]string)
 			}
+			const (
+				specialNetTag = "specialNet"
+				pciAddress    = "0000:08:00.0"
+			)
 			vmi.Annotations[v1.InstancetypeAnnotation] = testInstancetype
```

Comment:
Please consider moving the consts definition bellow the annotation assignment, as it breaks an existing logical flow:
```suggestion
			vmi.Annotations[v1.InstancetypeAnnotation] = testInstancetype
			const (
				specialNetTag = "specialNet"
				pciAddress    = "0000:08:00.0"
			)
```
BTW, the annotation definition could be replaced by `libvmi.WithLabel`, thus removing additional logic from the actual test.